modules/ROOT/pages/endpoint-https-receive.adoc

```diff
@@ -40,28 +40,59 @@ Possible values include:
-|
+|Yes
 |===
 
-== HTTP or HTTPS URL
+== HTTP or HTTPS Receive Endpoint URL
+
+=== Deploy to CloudHub on Public Port
+
+When an HTTP or HTTPS receive endpoint is deployed with the private port configuration enabled, the endpoint application is deployed on the public HTTPS port (8082), with the traffic routed through the xref:runtime-manager::dedicated-load-balancer-tutorial#shared-load-balancers [shared load balancer]. The value in the URL field is automatically generated when you deploy a message flow that uses this endpoint to CloudHub.
+
+You can view this URL by selecting the receiving endpoint from your deployed message flow or by clicking the endpoint name from the endpoints list. You can share this URL with the partners that send HTTP or HTTPS messages to this endpoint.
+
+If your organization uses CloudHub VPC without a dedicated load balancer (DLB), your organization’s platform administrator can configure the necessary xref:runtime-manager::vpc-firewall-rules-concept.adoc[VPC firewall rules] to allow specific IP addresses of your
+partners to access the endpoint.
+
+=== Deploy to CloudHub on a Private Port
+
+When an HTTP or HTTPS receive endpoint is deployed with the private port configuration enabled, the endpoint is deployed to the private HTTPS port (8092) and the internal URL of the endpoint appears on the the *Message Flow* or *Endpoint* detail page.
+
+After deploying the first message flow using the HTTP or HTTPS receive endpoint configuration, you can view the runtime application name for the endpoint from the *Endpoint* details page or from the *Receive* section of your message flow. To configure xref:runtime-manager/lb-mapping-rules[URL mapping rules] in the xref:runtime-manager::cloudhub-dedicated-load-balancer.adoc[DLB] to forward HTTP or HTTPS messages received from your backend applications to outbound B2B message flows, provide the application name to your organization’s platform administrator.
+
+If your backend processes consuming the HTTP or HTTPS receive endpoints are also running inside the same VPC, they can call the internal URL directly.
+
+If your backend processes consuming the endpoint are in a different network, your organization’s platform administrator can update DLB settings to xref:runtime-manager::lb-whitelists.adoc[whitelist] specific IP addresses to access the endpoint.
+
+The external URL that you can share with your partners is `https://{DLB-domain}/{Input-path}/{base-path}/{message-type}`.
+
+For example, if the name of your DLB domain is `mythical.lb.anypointdns.net`, the runtime application name is `b2b-inbound-http-ognq`, and the base path you configured in the HTTPS receive endpoint is `send2`, then your administrator adds a URL mapping rule in the DLB settings to forward incoming requests to the runtime application within your VPC:
+
+image::URL-mapping-rules-http.png[URL mapping rules in the DLB]
+
+With these configurations:
```

Comment:
I missed this: we don't want to continue a text that introduces a graphic after the graphic as well. Reword to create a sentence intro.

Reply:
Done